tex/homology/singular.tex

```diff
@@ -624,6 +646,52 @@ \subsection{Geometry of chain complexes}
 Indeed, one can check (just by writing down several $\sum$ signs)
 that the above identity holds.
 
+As a picture:
```

Comment:
> Save some readers' time (at the cost of writer's time).

Good habit for writers to get into ;)

Reply:
Side note, the picture doesn't look quite good because ∂ₓ(σ) consist of two disjoint points, and it's not clear that the arrows are meant to point to the "union". But I can't really see a way to make it clearer...?